src/components/dialogs/nuxs/index.tsx

```diff
@@ -34,24 +42,24 @@ const queuedNuxs: {
   }) => boolean
 }[] = [
   {
-    id: Nux.BookmarksAnnouncement,
+    id: Nux.FindContactsAnnouncement,
     enabled: ({currentProfile}) => {
       return isExistingUserAsOf(
-        '2025-09-08T00:00:00.000Z',
+        '2025-12-16T00:00:00.000Z',
```

Comment:
Should move the `isNative` check here, otherwise it's gonna technically activate on web even if it doesn't render.